HippUnfoldHCP/HippUnfoldHCP.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+set -eu
+pipedirguessed=0
+if [[ "${HCPPIPEDIR:-}" == "" ]]
+then
+  # pipedirguessed=1
+   #fix this if the script is more than one level below HCPPIPEDIR
+   export HCPPIPEDIR="$(dirname -- "$0")/.."
+fi
+
+source "$HCPPIPEDIR/global/scripts/newopts.shlib" "$@"
+source "$HCPPIPEDIR/global/scripts/debug.shlib" "$@"
+
+
+opts_SetScriptDescription "Make some BIDS structures and run HippUnfold"
+
+opts_AddMandatory '--study-folder' 'StudyFolder' 'path' "folder containing all subjects"
+opts_AddMandatory '--subject' 'Subject' 'subject ID' ""
+opts_AddOptional '--hippunfold-dir' 'HippUnfoldDIR' 'path' "location of HippUnfold outputs"
+
+opts_ParseArguments "$@"
+
+if ((pipedirguessed))
+then
+    log_Err_Abort "HCPPIPEDIR is not set, you must first source your edited copy of Examples/Scripts/SetUpHCPPipeline.sh"
+fi
+
+opts_ShowValues
+
+T1wFolder="$StudyFolder/$Subject/T1w"       # input data
+
+if [ -z ${HippUnfoldDIR} ] ; then
+  HippUnfoldDIR="${T1wFolder}/HippUnfold"
+fi 
+
+#Couldn't get non-BIDS to work (wouldn't do anything)
+#HippUnfoldT1wFolder="$HippUnfoldDIR/T1w"
+#HippUnfoldT2wFolder="$HippUnfoldDIR/T2w"
+#HippUnfoldT2wFolder="$HippUnfoldDIR/T1wT2w"
+
+HippUnfoldT1wFolder="$HippUnfoldDIR/T1w/sub-${Subject}/anat"
+HippUnfoldT2wFolder="$HippUnfoldDIR/T2w/sub-${Subject}/anat"
+HippUnfoldT1wT2wFolder="$HippUnfoldDIR/T1wT2w/sub-${Subject}/anat"
+
+HippUnfoldT1wFolderOut="$HippUnfoldDIR/T1w_hippunfold"
+HippUnfoldT2wFolderOut="$HippUnfoldDIR/T2w_hippunfold"
+HippUnfoldT1wT2wFolderOut="$HippUnfoldDIR/T1wT2w_hippunfold"
+
+
+T1wImage="$T1wFolder/T1w_acpc_dc_restore.nii.gz"
+T2wImage="$T1wFolder/T2w_acpc_dc_restore.nii.gz"
+
+
+if [ ! -f "$T1wImage" ]; then
+    echo "Error: T1w image not found at $T1wImage" >&2
+    exit 1
+fi
+
+if [ ! -f "$T2wImage" ]; then
+    echo "Error: T2w image not found at $T2wImage" >&2
+    exit 1
+fi
+
+
+mkdir -p "$HippUnfoldT1wFolder" "$HippUnfoldT2wFolder" "$HippUnfoldT1wT2wFolder"
+
+#Couldn't get non-BIDS to work (wouldn't do anything)
+#cp "$T1wImage" "$HippUnfoldT1wFolder/${Subject}_T1w_acpc_dc_restore.nii.gz"
+#cp "$T2wImage" "$HippUnfoldT1wFolder/${Subject}_T2w_acpc_dc_restore.nii.gz"
+#cp "$T1wImage" "$HippUnfoldT2wFolder/${Subject}_T1w_acpc_dc_restore.nii.gz"
+#cp "$T2wImage" "$HippUnfoldT2wFolder/${Subject}_T2w_acpc_dc_restore.nii.gz"
+#cp "$T1wImage" "$HippUnfoldT1wT2wFolder/${Subject}_T1w_acpc_dc_restore.nii.gz"
+#cp "$T2wImage" "$HippUnfoldT1wT2wFolder/${Subject}_T2w_acpc_dc_restore.nii.gz"
+
+cp "$T1wImage" "$HippUnfoldT1wFolder/sub-${Subject}_T1w.nii.gz"
+cp "$T2wImage" "$HippUnfoldT1wFolder/sub-${Subject}_T2w.nii.gz"
+cp "$T1wImage" "$HippUnfoldT2wFolder/sub-${Subject}_T1w.nii.gz"
+cp "$T2wImage" "$HippUnfoldT2wFolder/sub-${Subject}_T2w.nii.gz"
+cp "$T1wImage" "$HippUnfoldT1wT2wFolder/sub-${Subject}_T1w.nii.gz"
+cp "$T2wImage" "$HippUnfoldT1wT2wFolder/sub-${Subject}_T2w.nii.gz"
```

Comment:
Does it really need to create `T1w/T2w.nii.gz` and `T2w/T1w.nii.gz`?  Do the 3 calls even need 3 separate input folders (with currently the same initial contents)?  Why are there 3 hippunfold calls anyway?

Reply:
Matt says the T1w image in the T2w folder may be needed for registration.  The T2w image in the T1w folder still seems strange to me.  Another question, can these be symlinks instead of actual copies?  It looks like the container calls bind mount the entire study folder, so I would think relative symlinks would work.

Reply:
Done